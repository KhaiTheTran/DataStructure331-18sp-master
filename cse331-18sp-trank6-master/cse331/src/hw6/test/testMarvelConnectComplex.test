# Test load graph and BFS

#Load graph
LoadGraph marveNodeComplexGraph marveNodeComplexGraph.tsv

# check graph load
ListNodes marveNodeComplexGraph
ListChildren marveNodeComplexGraph LIBRA/GUSTAV
ListChildren marveNodeComplexGraph BRANDT
ListChildren marveNodeComplexGraph JONES,_RICHARD_MILHO

# Test the order of BFS
# Test BFS shortest path from LIBRA/GUSTAV to BRANDT
FindPath marveNodeComplexGraph LIBRA/GUSTAV BRANDT

# Test BFS shortest path from BRANDT to LIBRA/GUSTAV
FindPath marveNodeComplexGraph BRANDT LIBRA/GUSTAV

# Test BFS shortest path from BRANDT to JONES,_RICHARD_MILHO
FindPath marveNodeComplexGraph BRANDT JONES,_RICHARD_MILHO

# Test BFS shortest path from JONES,_RICHARD_MILHO to BRANDT
FindPath marveNodeComplexGraph JONES,_RICHARD_MILHO BRANDT

# Test BFS shortest path from JONES,_RICHARD_MILHO to LIBRA/GUSTAV
FindPath marveNodeComplexGraph JONES,_RICHARD_MILHO LIBRA/GUSTAV

# Test BFS shortest path from LIBRA/GUSTAV to JONES,_RICHARD_MILHO
FindPath marveNodeComplexGraph LIBRA/GUSTAV JONES,_RICHARD_MILHO