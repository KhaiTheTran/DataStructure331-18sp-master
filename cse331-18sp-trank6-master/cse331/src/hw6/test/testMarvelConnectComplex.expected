# Test load graph and BFS

#Load graph
loaded graph marveNodeComplexGraph

# check graph load
marveNodeComplexGraph contains: BRANDT JONES, RICHARD MILHO LIBRA/GUSTAV
the children of LIBRA/GUSTAV in marveNodeComplexGraph are: JONES, RICHARD MILHO(AVF 1) JONES, RICHARD MILHO(AVF 2)
the children of BRANDT in marveNodeComplexGraph are: JONES, RICHARD MILHO(AVF 3) JONES, RICHARD MILHO(AVF 4)
the children of JONES, RICHARD MILHO in marveNodeComplexGraph are: BRANDT(AVF 3) BRANDT(AVF 4) LIBRA/GUSTAV(AVF 1) LIBRA/GUSTAV(AVF 2)

# Test the order of BFS
# Test BFS shortest path from LIBRA/GUSTAV to BRANDT
path from LIBRA/GUSTAV to BRANDT:
LIBRA/GUSTAV to JONES, RICHARD MILHO via AVF 1
JONES, RICHARD MILHO to BRANDT via AVF 3

# Test BFS shortest path from BRANDT to LIBRA/GUSTAV
path from BRANDT to LIBRA/GUSTAV:
BRANDT to JONES, RICHARD MILHO via AVF 3
JONES, RICHARD MILHO to LIBRA/GUSTAV via AVF 1

# Test BFS shortest path from BRANDT to JONES,_RICHARD_MILHO
path from BRANDT to JONES, RICHARD MILHO:
BRANDT to JONES, RICHARD MILHO via AVF 3

# Test BFS shortest path from JONES,_RICHARD_MILHO to BRANDT
path from JONES, RICHARD MILHO to BRANDT:
JONES, RICHARD MILHO to BRANDT via AVF 3

# Test BFS shortest path from JONES,_RICHARD_MILHO to LIBRA/GUSTAV
path from JONES, RICHARD MILHO to LIBRA/GUSTAV:
JONES, RICHARD MILHO to LIBRA/GUSTAV via AVF 1

# Test BFS shortest path from LIBRA/GUSTAV to JONES,_RICHARD_MILHO
path from LIBRA/GUSTAV to JONES, RICHARD MILHO:
LIBRA/GUSTAV to JONES, RICHARD MILHO via AVF 1